index.html

```diff
@@ -49,12 +49,136 @@ <h1 id="nm">I’m Mariane<br><br>
+                            <li>&bull;&nbsp;&nbsp; 2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                    <ul class="tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li class="js">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div>
+                    <img src="./assets/Snapshoot Portfolio (1).png" alt="">
+                    <h2>Tonic</h2>
+                    <div class="details">
+                        <h3><b>CANOPY</b></h3>
+                        <ul>
+                            <li>&bull;&nbsp;&nbsp; Back End Dev</li>
+                            <li>&bull;&nbsp;&nbsp; 2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                    <ul class="tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li class="js">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div>
+                    <img src="./assets/Snapshoot Portfolio (2).png" alt="">
+                    <h2>Multi-Post<br>
+                        Stories</h2>
+                    <div class="details">
+                        <h3><b>CANOPY</b></h3>
+                        <ul>
+                            <li>&bull;&nbsp;&nbsp; Back End Dev</li>
+                            <li>&bull;&nbsp;&nbsp; 2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                    <ul class="tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li class="js">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="about">
+            <div>
+                <h2>About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+                <a href="#" class="lnk">LET’S CONNECT</a>
+                <div class="socials2">
+                    <img src="./assets/Frame 1.png" alt="Socials">
+                </div>
```

Comment:
- [ ] For the social media icons, Kindly replace this image with each icon put separately so you can add your social media handles to them like this 👇 

```
 <ul>
        <li><a href="https://twitter.com/your_handle"><i class="fa-brands fa-twitter fa-2x"></i></a></li>
        <li><a href="https://www.linkedin.com/in/your_handle"><i class="fa-brands fa-linkedin fa-2x"></i></a></li>
         .....
 </ul>
```

note: you can replace the font awesome icons with images 😄